packages/react-test-renderer/README.md

```diff
@@ -4,6 +4,8 @@ This package provides an experimental React renderer that can be used to render
 
 Essentially, this package makes it easy to grab a snapshot of the "DOM tree" rendered by a React DOM or React Native component without using a browser or jsdom.
 
+[https://reactjs.org/docs/test-renderer.html](https://reactjs.org/docs/test-renderer.html)
```

Comment:
Minor thing, but what do you think about adding a section like `update` below:

```
Documentation:

[https://reactjs.org/docs/test-renderer.html](https://reactjs.org/docs/test-renderer.html)
```

Reply:
cool! absolutely